migrations/20220428092250_initial_tables.js

```diff
@@ -17,9 +17,27 @@ exports.up = async (knex) => {
 
     def.primary(['id'])
   })
+
+  await knex.schema.createTable('recipe', (def) => {
```

Comment:
there's a README that describes tables in the db to be updated too
https://github.com/digicatapult/inteli-api/blob/main/docs/db.md